microsoft/surface/common/kernel/default.nix

```diff
@@ -6,6 +6,7 @@ let
 in {
   imports = [
     ./linux-6.10.x
```

Comment:
```suggestion
```

Reply:
@mexisme is removing the old kernel the only concern?

Reply:
> @mexisme is removing the old kernel the only concern?

That was my only thought, and it wasn't an especially big thought, TBQH.
The code otherwise looked pretty good.